packages/api/utils/ipLogging.test.ts

```diff
@@ -0,0 +1,22 @@
+import middy from "@middy/core";
+import { Context } from "aws-lambda";
+import { validRequest } from "../portfolioDrafts/commonPortfolioDraftMockData";
+import { mockPortfolioStep } from "../portfolioDrafts/portfolio/commonPortfolioMockData";
+import { IpCheckerMiddleware } from "./ipLogging";
+
+describe("Test console logging", function () {
+  it("should console log the event", async () => {
+    const request: any = {
+      validRequest,
+      body: mockPortfolioStep,
+      requestContext: { identity: { sourceIp: "10.2.2.2" } },
+    };
+    const baseHandler = (event: any) => {
+      return event;
+    };
+    const consoleLogSpy = jest.spyOn(console, "log");
```

Comment:
If log level is changed to `info`, spy must be updated, here and in similar places in other unit tests
```suggestion
    const consoleLogSpy = jest.spyOn(console, "info");
```